### DISCUSSION TITLE
Please help! Getting only one character as output.

### DISCUSSION THREAD

**Discussion by Pelad0x0** [openai/whisper#1688] [Q&A] [answered]
Hello! I'm trying to use the cli and I'm getting a wrong output.
I'm using Proxmox and a Lxc container (Debian 12) with a GPU (Gforce 3060ti) passthrough. Then compiled python 3.11 and installed whisper with /opt/py311/bin/python3 -m pip install git+https://github.com/openai/whisper.git.

When using `/home/sysoxo/.local/bin/whisper --model small --language es  --word_timestamps True  --output_format srt Trim.mp4`
the output is correct:
![image](https://github.com/openai/whisper/assets/117691049/5a4d34e6-75ae-4175-9cf9-5b232bc6f3b7)

The issue comes when I try to use the `medium` model `/home/sysoxo/.local/bin/whisper --model medium --language es  --word_timestamps True  --output_format srt Trim.mp4` I'm getting this output:
![image](https://github.com/openai/whisper/assets/117691049/a54f3065-7bba-4194-8626-4fb42dbcb450)

nvidia-smi output:
![image](https://github.com/openai/whisper/assets/117691049/43674ebe-ccf9-4c3f-bd1e-3aa13f1db965)

What I'm missing? Thank you very much.
Martin.

**Comment by glangford** [comment]
Note the difference between what is displayed on the console (which is just a log) compared to what is output in the actual .srt file. Also, try doing the same thing with the `medium` model without `--word_timestamps`.

**Reply by Pelad0x0** [reply]
I've tried without `--word_timestamps` with the same result. And the content of .srt file is the same as the output. Only `i`.
Thanks for the response.

**Reply by glangford** [reply]
That's unfortunate, what you are seeing is probably hallucination that causes a 'stutter' in the transcript. See for example these discussions

- https://github.com/openai/whisper/discussions/679
- https://github.com/openai/whisper/discussions/919

**Reply by Pelad0x0** [reply, marked as answer]
Thank you glangford! Those links lead me to the solution. I've added `--logprob_threshold None` to the command.

`whisper --task transcribe --model medium --language es --output_format srt --word_timestamps True --logprob_threshold None Trim.mp4`
